airbyte-integrations/bases/base-normalization/integration_tests/test_normalization.py

```diff
@@ -334,15 +332,39 @@ def dbt_test(destination_type: DestinationType, test_resource_name: str, test_ro
         destination_type,
         replace_identifiers,
     )
-    assert run_check_command(["dbt", "test", "--profiles-dir=.", "--project-dir=."], test_root_dir)
+    assert run_check_dbt_command("test", test_root_dir)
 
 
-def run_check_command(commands: List[str], cwd: str) -> bool:
+def run_check_dbt_command(command: str, cwd: str) -> bool:
     """
     Run dbt subprocess while checking and counting for "ERROR", "FAIL" or "WARNING" printed in its outputs
     """
     error_count = 0
+    commands = [
```

Comment:
We're trying to move away from calling docker in docker. This is creating a new level of inception.

Why can't you call dbt directly?

Reply:
NVM! it is a test :D

Reply:
the test is not run in a docker yes so it's not docker in docker

not calling it directly here so we don't have a direct dependency on it